diff --git a/analytics.js b/analytics.js
index 00c7700..9b3f124 100644
--- a/analytics.js
+++ b/analytics.js
@@ -5670,7 +5670,8 @@ function object(obj, strict) {
   }
   each(obj, function(key, val) {
     if (isodate.is(val, strict)) {
-      obj[key] = isodate.parse(val);
+      //obj[key] = isodate.parse(val);
+      obj[key] = val;
     } else if (type(val) === 'object' || type(val) === 'array') {
       traverse(val, strict);
     }
@@ -5696,7 +5697,8 @@ function array(arr, strict) {
     if (type(val) === 'object') {
       traverse(val, strict);
     } else if (isodate.is(val, strict)) {
-      arr[x] = isodate.parse(val);
+      //arr[x] = isodate.parse(val);
+      arr[x] = val;
     }
   });
   return arr;
@@ -14152,4 +14154,4 @@ module.exports={
 }
 
 },{}]},{},[1])(1)
-});
\ No newline at end of file
+});
